ixmp4/data/db/optimization/variable/repository.py

```diff
@@ -65,7 +65,7 @@ def _add_column(
         self.columns.create(
             name=column_name,
             constrained_to_indexset=indexset.id,
-            dtype=pd.Series(indexset.elements).dtype.name,
```

Comment:
ALso this looks quite expensive for what it does, no idea how to avoid it right now though...

Reply:
For all items like `parameter`, `variable`, etc, I'm thinking that tables similar to `IndexSetData` might relieve us of the whole `Column` table. Especially if we end up translating `parameter.data` etc to `IndexSetData.id`s, since the main purpose of the columns is currently to store which indexset the data columns belong to and which type it has. All of this can probably be taken care of in a better way, eliminating these function calls :)
However, this will only happen in a later PR, I'm afraid.